roles/kubernetes/node/tasks/install.yml

```diff
@@ -39,7 +13,16 @@
     ]"
   tags: facts
 
-- name: install | Install kubelet launch script
-  template: src=kubelet-container.j2 dest="{{ bin_dir }}/kubelet" owner=kube mode=0755 backup=yes
-  notify: restart kubelet
+- include: install_rkt.yml
```

Comment:
they both need `ssl_ca_dirs` https://github.com/kubernetes-incubator/kargo/pull/1015

i can move that fact setting to main.yml if that'd be better